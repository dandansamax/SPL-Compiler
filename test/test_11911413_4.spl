
int test(int a){
    return a;
}
int test2(int a){
    return a;
}
int test3(int a){
    return a;
}
int test_11911413_4(){
    int c 4;
    int a=0;
    a=test(test2(3test(a)));
   return 0;
}