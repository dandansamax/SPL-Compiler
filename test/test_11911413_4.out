Error type B at Line 12: Missing closing parenthesis '[' and ']'
Error type A at Line 14: unknown lexeme 3test
